6.6.3 and 6.7.0-rc start to emit a new warning that causes:
test_browsertab.py:13: Failure: Qt messages with level WARNING or above emitted

It still seems(?) to be able to find qutebrowser's installed dictionaries
fine and spellchecking works. Let's just ignore this warning for now.
--- a/pytest.ini
+++ b/pytest.ini
@@ -63,2 +63,4 @@
     Error in contacting registry: "org\.freedesktop\.DBus\.Error\.Disconnected" "Not connected to D-Bus server"
+    # Qt 6.6.3 and 6.7.0, not looked into but spellchecking still functions
+    ^Path override failed for key base::DIR_APP_DICTIONARIES and path .*
 xfail_strict = true
